src/components/Sidebar/Sidebar.minors.tsx

```diff
@@ -31,6 +32,20 @@ export function Item({
         {...props}
       />
     </Tip>
+  ) : (
+    <ItemLabeledStyled
+      aria-label={label}
```

Comment:
We shouldn't need the aria-label if the text is actually rendered inside the button. A screen reader will use that text. What we might want to do is add `aria-hidden` to the icon so it skips the icon and just reads the next.

Reply:
agreed. Although I think it may be up to the implementor to add the `aria-hidden` to the icon, the icon props at this juncture are readonly.